recipe/bld.bat

```diff
@@ -6,17 +6,24 @@ SET QUARTO_PANDOC=%LIBRARY_BIN%\pandoc.exe
 SET QUARTO_ESBUILD=%LIBRARY_BIN%\esbuild.exe
 SET QUARTO_DART_SASS=%LIBRARY_BIN%\sass.exe
 
-:: This is patched in for conda. This is otherwise set as a constant in `configuration`
-SET "QUARTO_VERSION=%PKG_VERSION%"
+:: Alter the configuration file with a dynamic value containing the full
+:: package version (e.g. 1.3.340). The only thing allowed in this file is
+:: export statements with static assignments, so we use a combination of a
+:: patch to update the source code to remove the original assignment and a
+:: build-time update to place the dynamic build-time PKG_VERSION as a static
+:: value.
+:: More context: https://github.com/conda-forge/quarto-feedstock/pull/7
+echo "export QUARTO_VERSION=%PKG_VERSION%" >> configuration
```

Comment:
Sorry, I gave you bad bat syntax

```suggestion
echo set "QUARTO_VERSION=%PKG_VERSION%" >> configuration
```

Reply:
That was me with the bad syntax :) I thought we would want to put an `export` in the file because `configure.cmd` would then change it to a `set`?